tests/st/calicoctl/test_crud.py

```diff
@@ -378,6 +378,10 @@ def test_namespace(self, data, namespaced):
         else:
             rc.assert_error(NOT_NAMESPACED)
 
+        # The label will be updated with the new namespace 
+        if kind == "WorkloadEndpoint" and namespaced:
+            data2['metadata']['labels']['projectcalico.org/namespace'] = "default"
```

Comment:
don't we also have the orchestrator label too?

Reply:
workload endpoint is always namespaced

Reply:
ah i see we're tweaking the namespaces in the test.

ok from what I can see the namespace for data2 should be "namespace2"

Reply:
will fix

Reply:
it feels like we should just update the data1 and data2 to have what is specified in the Metadata and Spec.

```
data1['metadata']['labels']['projectcalico.org/orchestrator'] = data1['spec']['orchestrator']
data1['metadata']['labels']['projectcalico.org/namespace'] = data1['metadata']['namespace']
data2['metadata']['labels']['projectcalico.org/orchestrator'] = data2['spec']['orchestrator']
data2['metadata']['labels']['projectcalico.org/namespace'] = data2['metadata']['namespace']
```

This would allow us to put in some more test data to check that the values in the supplied data are being overwritten with the correct values.